proto/proto/wallet.proto

```diff
@@ -4,58 +4,29 @@ package penumbra.wallet;
 import "transaction.proto";
 import "crypto.proto";
 
+// The wallet protocol is used by a wallet client, who wants to do some
+// transaction-related actions, to request data from a wallet service, which is
+// responsible for synchronizing and scanning the public chain state with one or
+// more full viewing keys.
+//
+// Wallet protocol requests include a hash of the full viewing key, used to
+// identify which set of data to query.  This also works as a pseudo-auth system
+// (assuming transport security, the client has to know the FVK to request its
+// data).  (TODO: refine this)
```

Comment:
The idea behind including a hash of the FVK is:

(1) it means that the protocol doesn't bake in the assumption that there's only one FVK per wallet service, even if we make that assumption in the current implementation;

(2) it's marginally better than sending the FVK as a way to identify the account data, since it doesn't reveal the full viewing key itself.  However, it's not *that* much better, since someone who learns the FVK hash (e.g., by being on the other end of a misconfigured server, by snooping on unencrypted network traffic, etc) can use the FVK hash as a bearer token to make requests to the wallet service (if they know which service to access).

Reply:
(To be clear, we should think of this as a first-pass mechanism, and figure out what a reasonable auth story looks like later. [Some ideas are here](https://twitter.com/FiloSottile/status/1520532532139339776).)